modules/orchestrator/src/main/java/org/eclipse/xpanse/orchestrator/OrchestratorService.java

```diff
@@ -67,6 +75,9 @@ public class OrchestratorService {
     @Resource
     private DeployVariableValidator deployVariableValidator;
 
+    @Value("${monitor.data.agent.enable}")
+    private Boolean monitorAgentEnabled;
+
```

Comment:
In case the configuration item is empty in the configuration file，we can use @Value("${monitor.data.agent.enable:false}") to set a default value .

Reply:
ok